test/utils.jl

```diff
@@ -97,6 +97,7 @@ end
     @test reo_s == s
     @test dims(reo_s) == dims(s)
 
+    @test_throws MethodError reorder(da, rand(2,3))
```

Comment:
```suggestion
```

Is this needed here? seems unrelated

Reply:
This is unrelated to the fixes of rand but I actual started this PR to work on test coverage therefore it ended up in there. I actually misunderstood how the dispatch in reorder works. Therefore yes we can just remove it.